content/zh/docs/hertz/tutorials/basic-feature/middleware/access-log.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "访问日志"
+date: 2023-03-14
+weight: 10
+description: 
+
+---
+访问日志可以收集所有 HTTP 请求的详细信息，包括时间、端口、请求方法等。Hertz 也提供了 access log 的[实现](https://github.com/hertz-contrib/logger)，这里的实现参考了 [fiber](https://github.com/gofiber/fiber/tree/master/middleware/logger)。
+
+## 安装
+
+```shell
+go get github.com/hertz-contrib/logger/accesslog
+```
+
+## 示例代码
+
+```go
+package main
+
+import (
+	"context"
+
+	"github.com/cloudwego/hertz/pkg/app"
+	"github.com/cloudwego/hertz/pkg/app/server"
+	"github.com/cloudwego/hertz/pkg/common/utils"
+	"github.com/hertz-contrib/logger/accesslog"
+)
+
+func main() {
+	h := server.Default(
+		server.WithHostPorts(":8080"),
+	)
+	h.Use(accesslog.New())
+	h.GET("/ping", func(ctx context.Context, c *app.RequestContext) {
+		c.JSON(200, utils.H{"msg": "pong"})
+	})
+	h.Spin()
+}
+```
+## 配置
+
+用户可以通过自定义初始化配置来设置访问日志的格式以及内容。
+
+### WithFormat
+使用 `WithFormat` 自定义日志格式
+
+示例代码:
+
+```go
+package main
+
+import (
+	"context"
+
+	"github.com/cloudwego/hertz/pkg/app"
+	"github.com/cloudwego/hertz/pkg/app/server"
+	"github.com/cloudwego/hertz/pkg/common/utils"
+	"github.com/hertz-contrib/logger/accesslog"
+)
+
+func main() {
+	h := server.Default(
+		server.WithHostPorts(":8080"),
+	)
+	h.Use(accesslog.New(accesslog.WithFormat("[${time}] ${status} - ${latency} ${method} ${path} ${queryParams}")))
+	h.GET("/ping", func(ctx context.Context, c *app.RequestContext) {
+		c.JSON(200, utils.H{"msg": "pong"})
+	})
+	h.Spin()
+}
+
+```
+
+### WithTimeFormat
+使用 `WithTimeFormat` 自定义时间格式.
+
```

Comment:
配置项的说明有点简略了，可以适当完善一下。可以再加上函数签名。

The description of the configuration items is a bit brief, and it can be improved appropriately. Function signatures can be added.